Tasks/ServiceFabricComposeDeploy/task.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "19C02B15-D377-40E0-9EFA-3168506E0933",
     "name": "ServiceFabricComposeDeploy",
-    "friendlyName": "Service Fabric Compose Deploy",
+    "friendlyName": "Service Fabric Compose Deploy (PREVIEW)",
```

Comment:
We can't make it preview once it has gone out of preview. Also, this task has been shipped in on-premise TFS release and hence should not be preview

Reply:
It never should have gone out of preview, someone else made that change with out reference to the fact that the platform feature is still in preview.

Reply:
It was taken out of preview as a task should not be in preview when it is shipping in on-prem TFS. Can we make assumption that task itself is stable enough to be non-preview regardless of target platform?
@kmkumaran do you have a suggestion here?